packages/common/src/bag3d/common/resources/__init__.py

```diff
@@ -101,24 +110,41 @@
 }
 
 RESOURCES_PROD = {
-    "gdal": gdal_local,
+    "gdal": gdal,
     "file_store": file_store_gilfoyle,
     "file_store_fastssd": file_store_gilfoyle_fastssd,
     "db_connection": db_connection,
-    "pdal": pdal_local,
+    "pdal": pdal,
     "lastools": lastools,
     "tyler": tyler,
     "geoflow": geoflow,
     "roofer": roofer,
     "version": version,
 }
 
-# Resource definitions for import
+RESOURCES_DEFAULT = {
```

Comment:
This is for initialising Resources with Null values (all except the database resource and the VersionResource.